packages/client/tests/functional/engine-selection-logic/tests.ts

```diff
@@ -18,7 +18,7 @@ testMatrix.setupTestSuite(
     })
 
     describe('via env var', () => {
-      testIf(clientMeta.dataProxy /** = --no-engine */)(
+      testIf(clientMeta.dataProxy /** = --no-engine */ && !cliMeta.previewFeatures.includes('metrics'))(
```

Comment:
I don't understand why we need to check that `metrics` is not enabled here?

Reply:
It's not clear if you lack context indeed, related: https://github.com/prisma/prisma/pull/22430#discussion_r1452528050

Reply:
```suggestion
      # Note that when using Accelerate, the client will fail to init if metrics is enabled.
      # so we only run this tests in this file when `metrics` is not enabled.
      testIf(clientMeta.dataProxy /** = --no-engine */ && !cliMeta.previewFeatures.includes('metrics'))(
```